Sources/Interaction/Manipulators/MouseManipulator/index.js

```diff
@@ -28,7 +29,7 @@ const DEFAULT_VALUES = {
   shift: false,
   control: false,
   alt: false,
-  pinch: false,
+  scroll: false,
 };
```

Comment:
As I told you, since you are refactoring it, pinch or scroll are not states to test against. They are more a mode that you want to be enabled against. Therefore, you can not keep the same naming. Maybe `enableMouseDrag`, `enableScroll` and `enablePinch`? Where by default all are false except `enableMouseDrag`...

Reply:
this PR is already huge, can we keep that for later? So that the behavior just doesn't change?

Reply:
ok, do you want to merge to a branch `next` and create new PR. Then when we feel that `next` is ready we can merge it to master. Or do you rather merge that PR into master before doing any other work/PR?